parts/windows/windowscsehelper.ps1

```diff
@@ -426,19 +426,6 @@ function Get-WindowsVersion {
     }
 }
 
-function Get-WindowsPauseVersion {
```

Comment:
So:
* This file is included in the script produced by AgentBaker
* The functions in it are used by scripts included in the VHD
* So removing this function means that AgentBaker will not be able to provision older windows VHDs. 
* It also won't be able to provision the current windows VHD until we rollout a new VHD

So let's leave this function here please.